docs/adding-a-page.md

```diff
@@ -98,7 +98,7 @@ export default (
 
 ### Starting the server
 
-Now that the component is setup we can run `npm start dev:search` and navigate to
+Now that the component is setup we can run `yarn dev:search` and navigate to
```

Comment:
This whole guide is based around the search app. I'm not going to spend time on updating it for this patch. TBH I was just replacing all instances of `npm run`.